src/main/java/com/example/solidconnection/mentor/controller/MentoringController.java

```diff
@@ -0,0 +1,85 @@
+import com.example.solidconnection.siteuser.domain.Role;
+import com.example.solidconnection.siteuser.domain.SiteUser;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/mentorings")
+public class MentoringController {
+
+    private final MentoringCommandService mentoringCommandService;
+    private final MentoringQueryService mentoringQueryService;
+
+    @PostMapping("/apply")
+    public ResponseEntity<MentoringApplyResponse> applyMentoring(
+            @AuthorizedUser SiteUser siteUser,
+            @Valid @RequestBody MentoringApplyRequest mentoringApplyRequest
+    ) {
+        MentoringApplyResponse response = mentoringCommandService.applyMentoring(siteUser.getId(), mentoringApplyRequest);
+        return ResponseEntity.ok(response);
+    }
+
+    @RequireRoleAccess(roles = {Role.ADMIN, Role.MENTOR})
+    @GetMapping("/apply")
+    public ResponseEntity<MentoringListResponse> getMentorings(
+            @AuthorizedUser SiteUser siteUser
+    ) {
+        MentoringListResponse responses = mentoringQueryService.getMentorings(siteUser.getId());
+        return ResponseEntity.ok(responses);
+    }
+
+    @RequireRoleAccess(roles = {Role.ADMIN, Role.MENTOR})
+    @PatchMapping("/{mentoring-id}/apply")
+    public ResponseEntity<MentoringConfirmResponse> confirmMentoring(
+            @AuthorizedUser SiteUser siteUser,
+            @PathVariable("mentoring-id") Long mentoringId,
+            @Valid @RequestBody MentoringConfirmRequest mentoringConfirmRequest
+    ) {
+        MentoringConfirmResponse response = mentoringCommandService.confirmMentoring(siteUser.getId(), mentoringId, mentoringConfirmRequest);
+        return ResponseEntity.ok(response);
+    }
+
+    @RequireRoleAccess(roles = {Role.ADMIN, Role.MENTOR})
+    @PatchMapping("/{mentoring-id}/check")
+    public ResponseEntity<MentoringCheckResponse> checkMentoring(
+            @AuthorizedUser SiteUser siteUser,
+            @PathVariable("mentoring-id") Long mentoringId
+    ) {
+        MentoringCheckResponse response = mentoringCommandService.checkMentoring(siteUser.getId(), mentoringId);
+        return ResponseEntity.ok(response);
+    }
+
+    @RequireRoleAccess(roles = {Role.ADMIN, Role.MENTOR})
+    @GetMapping("/check")
+    public ResponseEntity<MentoringCountResponse> getNewMentoringsCount(
+            @AuthorizedUser SiteUser siteUser
```

Comment:
```suggestion
    public ResponseEntity<MentoringCountResponse> getUncheckedMentoringsCount(
            @AuthorizedUser SiteUser siteUser
```

New 보다는 "체크되지 않았음"을 직역하는 unchecked 어떤가요? 언어를 통일하는 관점에서요!

Reply:
동의합니다 !